changelog.md

```diff
@@ -4,21 +4,29 @@
 
 * Added Github CLI to gitpod environment for greater ease of use
 * Added `starport build` command to build and install app binaries.
+* Improved the first-time experience for readers of the Starport readme and parts of the Starport Handbook.
```

Comment:
Can we remove repeated lines on this file?

Reply:
Sure can